hack/rbacgenerator/function.js

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
Even if its a hack/temp thing, it better to not name the folder as hack. Adding a  doc file to explain the choice might be a better option.